File: stub-android/src/main/java/android/app/Activity.java
package android.app;

import android.support.annotation.IdRes;
import android.view.View;

import kotlin.NotImplementedError;

/**
 * @author Dmitry Borodin on 2017-10-14.
 */
public class Activity {

//	public View findViewById(@IdRes int id) {
//		throw new NotImplementedError("stub");
//	}

	public <T extends View> T findViewById(@IdRes int id) {
		throw new NotImplementedError("stub");
	}
}

File: stub-android/src/main/java/android/app/Activity.kt
package android.app

import android.content.Context
import android.support.annotation.IdRes
import android.view.View

/**
 * @author Dmitry Borodin on 2017-10-14.
 */
class Activity {

    //	public View findViewById(@IdRes int id) {
    //		throw new NotImplementedError("stub");
    //	}

    fun <T : View> findViewById(@IdRes id: Int): T = throw NotImplementedError("stub")

    val context: Context
        get() = throw NotImplementedError("stub")
}
